docs/metamaskChainConnection.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Configure MetaMask for All Supported Chains
```

Comment:
I dont think this should be under the root category , i think its best under the blockchain and web3 section